Userland/Utilities/network-settings.cpp

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023, Fabian Dellwing <fabian@dellwing.net>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <AK/JsonObject.h>
+#include <AK/JsonParser.h>
+#include <Kernel/API/POSIX/errno.h>
```

Comment:
Do you even need these includes?
I guess that you can rely on `LibCore/System.h` to provide them

Reply:
vscode said so, will try to remove them again.

Reply:
I was able to remove a few.